app/js/views/microphone.jsx

```diff
@@ -26,6 +27,15 @@ export default class Microphone extends React.Component {
     this.click = this.click.bind(this);
   }
 
+  shouldComponentUpdate(nextProps, nextState) {
+    if (!this.server.isLoggedIn) {
```

Comment:
Maybe a race condition here? On first render of the calendar view, I'm seeing an empty DOM node where the microphone should be rendered:
```HTML
<section class="microphone"><!-- react-empty: 1 --></section>
```